src/components/redux/categories/categories.js

```diff
@@ -0,0 +1,14 @@
+const initialState = [];
+
+export function status() {
+  return {
+    type: 'STATUS',
+  };
+}
+
+export default function reducer(state = initialState, action) {
+  if (action.type === 'STATUS') {
+    return 'under construction...';
+  }
```

Comment:
- [ ] [Optional] [Highly recommended] Same here, Please use arrow function syntax for all functions instead of regular function syntax.